model/Check/System/AwsRDSAcuUtilizationCheck.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2023 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ */
+
+namespace oat\taoSystemStatus\model\Check\System;
+
+use DateInterval;
+use DateTime;
+use oat\oatbox\reporting\Report;
+use oat\taoSystemStatus\model\Check\Traits\PieChartReportRenderer;
+use oat\taoSystemStatus\model\SystemCheckException;
+
+/**
+ * Class AwsRDSAcuUtilizationCheck
+ * @package oat\taoSystemStatus\model\Check\System
+ * @author Makar Sichevoi, <makar.sichevoy@taotesting.com>
+ */
+class AwsRDSAcuUtilizationCheck extends AbstractAwsRDSCheck
+{
+    use PieChartReportRenderer;
+
+    private const PARAM_PERIOD = 'period';
+    private const ID_ACU_UTILIZATION = "acuutilization";
+    private const NAMESPACE = 'AWS/RDS';
+    private const METRIC_NAME = 'ACUUtilization';
+
+    /**
+     * @inheritdoc
+     */
+    protected function doCheck(): Report
+    {
+        $rdsHost = $this->getRDSHost();
```

Comment:
**suggestion**: as we do not use the output of those functions anywhere here except getInstanceData, can we move those function calls there?

Reply:
fixed